core/src/main/java/org/infinispan/configuration/parsing/Parser80.java

```diff
@@ -1083,6 +1107,31 @@ protected void parseCacheElement(XMLExtendedStreamReader reader, Element element
             this.parseExpiration(reader, builder);
             break;
          }
+         case DATA_CONTAINER: {
+            parseDataContainer(reader, holder);
+            break;
+         }
+         case SECURITY: {
+            this.parseCacheSecurity(reader, builder);
+            break;
+         }
+         default: {
+            reader.handleAny(holder);
+         }
+      }
+   }
+
+   protected void parseFullCacheElement(XMLExtendedStreamReader reader, Element element, ConfigurationBuilderHolder holder) throws XMLStreamException {
```

Comment:
These are shared argument for any cache. I am not very happy about the name 'full' cache either - any suggestions?